web/src/lightbox.js

```diff
@@ -509,6 +509,12 @@ export function next() {
     $(".image-list .image.selected").next().trigger("click");
 }
 
+function remove_video_player() {
+    // Remove the video player from the DOM. Used when closing lightbox
+    // so that the video doesn't run in background.
+    $("#lightbox_overlay .video-player").html("");
+}
```

Comment:
This should be passed as an `on_close` hook to `open_overlay` rather than called in an ad-hoc fashion.